install/helm.go

```diff
@@ -228,7 +228,7 @@ func (k *K8sInstaller) generateManifests(ctx context.Context) error {
 	// Store all the options passed by --config into helm extraConfig
 	extraConfigMap := map[string]interface{}{}
 	for k, v := range deprecatedCfgOpts {
-		extraConfigMap[k] = v
+		extraConfigMap[k] = string(v)
```

Comment:
This means that `extraConfigMap[k]` will have a `string` value if `v` is a string, or the empty string otherwise. Is this the desired behavior? Alternative code code be:

```go
if s, ok := v.(string); ok {
    extraConfigMap[k] = s // Happy case: v is a string
} else {
    log.Warn("%k value %v is not a string, ignoring", k, v)
}
```